packages/fuel-indexer-lib/src/config/cli.rs

```diff
@@ -162,13 +162,13 @@ pub struct IndexerArgs {
     #[clap(long, help = "Number of seconds over which to allow --rate-limit-rps.")]
     pub rate_limit_window_size: Option<u64>,
 
-    /// Maximum length of time (in seconds) that an indexer's event handler can run before timing out.
+    /// The number of WASM opcodes after which the indexer's event handler will stop execution.
     #[clap(
         long,
-        default_value_t = defaults::INDEXER_HANDLER_TIMEOUT,
-        help = "Maximum length of time (in seconds) that an indexer's event handler can run before timing out."
+        help = "The number of WASM opcodes after which the indexer's event handler will stop execution.",
+        default_value_t = 30_000_000_000
     )]
-    pub indexer_handler_timeout: u64,
+    pub indexer_handler_metering_points: u64,
```

Comment:
- I like the descriptiveness of the naming here, but let's shorten this. Maybe `max_metering_points` ? 
  - The fact that these points are for an indexer handler are implied (and mentioned explicitly in the opt description)